### PR TITLE
Some prereqs for implementing the auth code grant.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
This PR has a bunch of _nonsense_ for getting the auth code grant set up in a future PR:

🎨 Configures our front-end build & includes Forge for pretty web views.

🏁 Remove the feature flag for OAuth since like _it's shiiiiipped_.

🌐 Adds `web` middleware group & stubs out routes & views for the auth code flow.

#### How should this be reviewed?
Tests should all still pass, and there shouldn't really be any user-facing changes besides a bunch of routes that now return `503`s and a couple of placeholder views.

#### Checklist
- [ ] Documentation added for changed endpoints.
- [ ] Tests added for new features/bug fixes.

---
For review: @angaither @weerd

### CODE REVIEW

File: app/Http/Controllers/WebController.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+class WebController extends Controller
+{
+    /**
+     * Make a new HomeController, inject dependencies,
+     * and set middleware for this controller's methods.
+     */
+    public function __construct()
+    {
+        // ...
```
Comment (weerd) [inline app/Http/Controllers/WebController.php]: to be continued?

Comment (DFurnes) [inline app/Http/Controllers/WebController.php]: Yep, nothing much going on here at the moment.

File: app/Http/Middleware/Authenticate.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ class Authenticate
      */
     public function handle($request, Closure $next)
     {
-        if (! Auth::check()) {
+        if (! auth()->check()) {
```
Comment (weerd) [inline app/Http/Middleware/Authenticate.php]: @DFurnes's one man mission to eradicate all facades!

**Comment by weerd** [comment]
👍

File: app/Http/Middleware/Authenticate.php (hunk 2 above)
Comment (DFurnes) [inline app/Http/Middleware/Authenticate.php]: 😏

**Pull request merged**